openg2p_registration/views/openg2p_registration.xml

```diff
@@ -22,7 +22,7 @@
             </field>
         </record>
 
-        <record id="openg2p_registration_view_tree_activity" model="ir.ui.view">
+        <!-- <record id="openg2p_registration_view_tree_activity" model="ir.ui.view">
```

Comment:
Why did we comment this code?

Reply:
It showed error while upgrading registration module.